app/controllers/topical_events_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
 
   def show
     @classification = TopicalEvent.friendly.find(params[:id])
-    @policies = @classification.published_policies.includes(:translations, :document)
+    @policies = @classification.published_policies
```

Comment:
I was wondering about that.  They are used in some counts I believe, and we'll be restoring them at some point.